pkg/kubectl/cmd/create_test.go

```diff
@@ -33,7 +33,8 @@ func TestExtraArgsFail(t *testing.T) {
 
 	f, _, _, _ := cmdtesting.NewAPIFactory()
 	c := NewCmdCreate(f, buf, errBuf)
-	if ValidateArgs(c, []string{"rc"}) == nil {
+	options := CreateOptions{}
+	if options.ValidateArgs(c, []string{"rc"}) == nil {
```

Comment:
Add more tests for `raw` as well.

Such as using `--raw` and `--edit` together.